appKey: 123456
appSecret: 123456
deviceId: 123456
deviceType: 123456
